src/minisat/core/SolverTypes.h

```diff
@@ -140,7 +140,8 @@ class Clause {
         unsigned learnt    : 1;
         unsigned has_extra : 1;
         unsigned reloced   : 1;
-        unsigned size      : 27; }                            header;
+        unsigned glue      : 3;
```

Comment:
I have had issues before where having too few bits for glues was causing issues. Some instances caused very high glues for all clauses, leading to the max (in this case, 7), assigned to all clauses, which consequently lead to extremely bad performance. This may not hit you at all, depending on the problems you are solving. So this is just a heads-up in case some instance is suddenly very slow/impossible to solve, it could be due to this.

I would suggest a few more bits. Even better, not bitstuffing it and using 16b/32b may be safer. But maybe I'm just paranoid :S